### ISSUE TITLE
show command misbehavior

### ISSUE THREAD

**Issue by aleprovencio** [gopasspw/gopass#1362]
### Summary
The `show` argument used to print the full contents of a secret, now it just copies the password to clipboard. This is a bigger problem with multiline secrets.

### Steps To Reproduce
``gopass show secret.gpg``

### Expected behavior
Secret should be printed on stdout

### Environment
- Archlinux
- gopass 1.9.2-1

**Comment by SalocinHB** [comment]
I'm seeing the same bug on macos 10.15. 

@aleprovencio as a workaround, you can use `gopass show -f secret.gpg`, that will print the secret instead of copying it, but I agree that this is pretty annoying.

**Comment by aleprovencio** [comment]
Good catch @SalocinHB, that prevents downgrading :)

**Comment by marco-m** [comment]
There is also another related bug: if you don't have `xsel` or `xclip`, you get a warning that mentions the non-existing flag `-p`:

```
$ gopass show foo/bar
WARNING: No clipboard available. Install xsel or xclip or use -p to print to console
```

and

```
gopass show -p foo/bar
Incorrect Usage: flag provided but not defined: -p

NAME:
   gopass show - Display a secret

USAGE:
   gopass show [command options] [arguments...]

DESCRIPTION:
   Show an existing secret and optionally put its first line on the clipboard. If put on the clipboard, it will be cleared after 45 seconds.

OPTIONS:
   --clip, -c        Copy the first line of the secret into the clipboard (default: false)
   --alsoclip, -C    Copy the first line of the secret and show everything (default: false)
   --qr              Print the first line of the secret as QR Code (default: false)
   --force, -f       Display the password even if safecontent is enabled (default: false)
   --password, -o    Display only the password (default: false)
   --sync, -s        Sync before attempting to display the secret (default: false)
   --revision value  Show a past revision
   --help, -h        show help (default: false)
```

@dominikschulz let me know if you want me to file a separate issue instead.

**Comment by klartext** [comment]
Thanks for reporting this. It's really annoying not to have the answer on stdout.
Also bad: I don't want the output be in the clipboard.
If by accident copying the clipboard to where it does not belong, that's quite bad.

**Comment by d1ee** [comment]
I have the same problem, but I just had to change to config and it works again.

```
# Somehow, autoclip config was true
$ gopass config autoclip
autoclip: true
```

```
$ gopass config autoclip false
```
OR
```
gopass config --store <storename> autoclip false
```

**Comment by dominikschulz** [comment]
The error message should be fixed now.

**Comment by klartext** [comment]
> The error message should be fixed now.

OK, fine. When will there be the next release, so that it can go into packages?

**Comment by dominikschulz** [comment]
This might take a while. We're changing a lot of stuff right now.
You can help to speed it up by testing the master branch.

**Comment by dominikschulz** [comment]
Fixed by #1513
